### DISCUSSION TITLE
Composite Config Files

### DISCUSSION THREAD

**Discussion by Iaphetes** [leftwm/leftwm#1224] [Q&A]
Hi everyone,

I've been looking into using LeftWM on my machines and syncing the configs between very different setups.
Since I'm also trying out the separate workspace feature, which is very hardware dependent, I was thinking, if it is possible to composite the config.ron file somehow.

I was thinking about some sort of include statement, where I include setup-specific parts of the configs (like the workspace list).

I know I could write a launch script, which does this, but is there a native way, I just haven't seen?

Thanks in advance!

**Comment by Eskaan** [comment]
No, we don't have this implemented, although there has been suggestions to support something like partial config reloading, which could also be applied in your case.

**Reply by Iaphetes** [reply]
Thanks. Is that discussed actively somewhere?

**Reply by Eskaan** [reply]
Not really atm, but @VuiMuich suggested it as a feature.
